api.go

```diff
@@ -1548,14 +1548,26 @@ func oAuthClientTokensHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	page := 1
```

Comment:
Here we are changing logic behind already public endpoint https://tyk.io/docs/tyk-rest-api/oauth-key-management/#retrieve-all-current-tokens-for-specified-oauth-client which I believe some of Tyk users already started using.

I think it might be better:
1. to provide a way here to retrieve all tokens and optional pagination so we won't break existing functionality
2. to add new endpoint for paginated tokens as clients will likely need pagination info to build a ruler

I'd vote for 2nd as it provides backward compatibility and gives more tooling to build pagination UI

Reply:
I'd go with the 2nd option too. I'd update this PR to reflect that

Reply:
Kindly take a look at https://github.com/TykTechnologies/tyk/pull/2109/commits/6b91b049204211be7df6505b4924ed57859d0525